tools/livecd-creator

```diff
@@ -250,6 +277,13 @@ def main():
         logging.error(u"%s creation failed: %s", options.image_type, e)
         return 1
 
+    if creator.releasever and int(creator.releasever) < 37:
+        creator.BIOSbooter = 'SYSLINUX'
```

Comment:
This is going to lead to incorrect results because non-Fedora distros that use this tool have different versioning schemes and also may not have SYSLINUX. For example, RHEL. It's probably just easier to use GRUB by default unless specified to use syslinux by the user, or just drop SYSLINUX support entirely.